content/applications/general/email_communication/email_domain.rst

```diff
@@ -152,12 +152,12 @@ also be used for other lesser known providers.
 Use a default email address
 ===========================
 
-To force the email address from which emails are sent, you need to create the following key in
-the System Parameters of the database:
+To force the email address from which emails are sent, you need to use a combination of the following keys in the system parameters of the database:
```

Comment:
``To force the email address from which emails are sent, a combination of the following keys needs to be set in the system parameters of the database:``

Reply:
Hey @tiku-odoo The whole file is in the second person so I wouldn't change that sentence.

Reply:
@jqu-odoo Thanks for the suggestion, the content team generally tries to keep things consistent by not writing in the 2nd person. I realize there are a few sentences where this isn't the case in our current documentation. We are working to get those edits made and the grammar corrected. In this situation I'd like to make this small change.